docs/image-redactor/index.md

```diff
@@ -1,22 +1,113 @@
+  instructions on how to install it for your operating system.
+
+  The Image-Redactor was tested with a minimal version of 4.0.0
 
-- Running in Python.
-- Running using Docker.
-- Customizing.
-- API reference.
+=== "Using pip"
+    
+    !!! note "Note"
+        Consider installing the Presidio python packages on a virtual environment like venv or conda.
+    
+    To get started with Presidio-image-redactor,
+    download the package and the `en_core_web_lg` spaCy model:
+    
+    ```sh
+    pip install presidio-image-redactor
+    python -m spacy download en_core_web_lg
+    ```
 
-### Running in Python
+=== "Using Docker"
+    
+    !!! note "Note"
+        This requires Docker to be installed. [Download Docker](https://docs.docker.com/get-docker/).
+    
+    ```sh
+    # Download image from Dockerhub
+    docker pull mcr.microsoft.com/presidio-image-redactor
+    
+    # Run the container with the default port
+    docker run -d -p 5003:5003 mcr.microsoft.com/presidio-image-redactor:latest
+    ```
 
-### Running using Docker
+=== "From source"
+    
+    First, clone the Presidio repo. [See here for instructions](../installation.md#install-from-source).
+    
+    Then, build the presidio-image-redactor container:
+    
+    ```sh
+    cd presidio-image-redactor
+    docker build . -t presidio/presidio-image-redactor
+    ```
 
-### Customizing
+## Getting started
+
+=== "Python"
+    
+    Once the Presidio-image-redactor package is installed, run this simple script:
+    
+    ```python
+    from PIL import Image
+    from presidio_image_redactor import ImageRedactorEngine
+    
+    # Get the image to redact using PIL lib (pillow)
+    image = Image.open("ocr_text.png")
+    
+    # Initialize the engine
+    engine = ImageRedactorEngine()
+    
+    # Redact the image with pink color
+    redacted_image = engine.redact(image, (255, 192, 203))
+    
+    # save the redacted image 
+    redacted_image.save("new_image.png")
+    # open the image for viewing
+    redacted_image.show()
+    
+    ```
+
+=== "As an HTTP server"
+    
+    You can run presidio image redactor as an http server using either python runtime or using a docker container.
+    
+    #### Using docker container
+    
+    ```sh
+    cd presidio-image-redactor
+    docker run -p 5003:5003 presidio-image-redactor 
+    ```
+    
+    #### Using python runtime
+    
+    !!! note "Note"
+        This requires the Presidio Github repository to be cloned.
+    
+    ```sh
+    cd presidio-image-redactor
+    python app.py
+    # use ocr_test.png as the image to redact, and 255 as the color fill. 
+    # out.png is the new redacted image received from the server.
+    curl -XPOST "http://localhost:3000/redact" -H "content-type: multipart/form-data" -F "image=@ocr_test.png" -F "data=\"{'color_fill':'255'}\"" > out.png
+    ```
+Python script example can be found under:
+/presidio/e2e-tests/tests/test_image_redactor.py
 
 ### API reference
```

Comment:
I think this header indentation is wrong. its currently under ##Getting started.

Reply:
![Screen Shot 2021-03-01 at 12 23 50](https://user-images.githubusercontent.com/1862328/109484531-37c99c80-7a89-11eb-997e-34da0f8b3cff.png)

Reply:
That's how it currently looks (it is aligned with anonymizer and analyzer).
How should I change it? :)

Reply:
Done.